src/Model/User.php

```diff
@@ -1460,11 +1464,20 @@ public static function create(array $data): array
 			Contact::updateSelfFromUserID($uid, true);
 		}
 
-		Hook::callAll('register_account', $uid);
+		$eventDispatcher = DI::eventDispatcher();
+
+		$hook_data = [
+			'uid' => $uid,
+		];
+
+		$eventDispatcher->dispatch(
+			new ArrayFilterEvent(ArrayFilterEvent::ACCOUNT_REGISTER, $hook_data),
```

Comment:
"Fire-and-Forget" event :)

Maybe something like `BroadcastEvent:ACCOUNT_REGISTERED` ?